sentry/values.yaml

```diff
@@ -933,7 +933,7 @@ clickhouse:
     imageVersion: "20.12.8.5"
     configmap:
       remote_servers:
-        internal_replication: true
+        internal_replication: false
```

Comment:
because this has been in incorrect place - it means that the default behaviour has been `false` for everyone. That is why this should be false, and also docs says that default value is false.